src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Kindly ensure you define add book and remove book following the [ducks pattern](https://github.com/erikras/ducks-modular-redux), it needs to be something like this ` const ADD_BOOK = bookstore/books/ADD_BOOK`